src/libraries/pkg/test/testPackages.proj

```diff
@@ -13,7 +13,9 @@
     <!-- support override via commandline -->
     <RuntimesToInclude Condition="'$(RuntimesToInclude)' != ''" Include="$(RuntimesToInclude)" />
     <TargetFrameworksToInclude Condition="'$(TargetFrameworksToInclude)' != ''" Include="$(TargetFrameworksToInclude)" />
-
+    <TestPackages Include="$(LibrariesAllConfigShippingPackagesDir)*.nupkg" />
+    <SymbolsPackages Include="$(LibrariesAllConfigShippingPackagesDir)*.symbols.nupkg" />
+    <TestPackages Remove="@(SymbolsPackages)" />
```

Comment:
Best practice is to exclude rather than include/remove.

Reply:
Exclude doesnt work correctly with some regexs.

Reply:
I'm simply suggesting to to reorder these two items like follows:
```
 <SymbolsPackages Include="$(LibrariesAllConfigShippingPackagesDir)*.symbols.nupkg" />
 <TestPackages Include="$(LibrariesAllConfigShippingPackagesDir)*.nupkg" Exclude="@(SymbolsPackages)" />
```